packages/oas/src/operation/index.ts

```diff
@@ -857,4 +857,32 @@ export class Callback extends Operation {
   }
 }
 
-export class Webhook extends Operation {}
+export class Webhook extends Operation {
+  /**
+   * OpenAPI API Definition that this webhook originated from.
+   */
+  declare api: RMOAS.OAS31Document;
```

Comment:
TypeScript threw an error here saying that the `api` property was going to overwrite the base `api` property in `Operation` so I had to add a `declare` here to mark it intentional.

Reply:
TIL